### PR TITLE
findbugs-exclude.xml: Remove suppressions for StashBuildTrigger

### PR DESCRIPTION
**Pull request by proski**
The issues in StashBuildTrigger have been fixed in #86

### CODE REVIEW

File: src/main/java/stashpullrequestbuilder/stashpullrequestbuilder/repackage/org/apache/commons/httpclient/contrib/ssl/EasySSLProtocolSocketFactory.java
Comment (jakub-bochenski) [inline src/main/java/stashpullrequestbuilder/stashpullrequestbuilder/repackage/org/apache/commons/httpclient/contrib/ssl/EasySSLProtocolSocketFactory.java]: I wouldn't want to change the copied code in any way, that is the reason to have findbugs exclude in the first place. 
I would rather leave the exception for `~stashpullrequestbuilder\.stashpullrequestbuilder\.repackage\..*` or wait until #68 is merged.

**Review by jakub-bochenski — CHANGES_REQUESTED** [review]

File: src/main/java/stashpullrequestbuilder/stashpullrequestbuilder/repackage/org/apache/commons/httpclient/contrib/ssl/EasySSLProtocolSocketFactory.java
Comment (proski) [reply src/main/java/stashpullrequestbuilder/stashpullrequestbuilder/repackage/org/apache/commons/httpclient/contrib/ssl/EasySSLProtocolSocketFactory.java]: OK, removing that part. It's a small PR now, but at least I know that the remaining suppressions are still needed.

Comment (proski) [reply src/main/java/stashpullrequestbuilder/stashpullrequestbuilder/repackage/org/apache/commons/httpclient/contrib/ssl/EasySSLProtocolSocketFactory.java]: Fixed

**Review by jakub-bochenski — APPROVED** [review]

**Pull request merged**
